en/tween/index.md

```diff
@@ -1,138 +1,19 @@
-# Tweening
+# Tween System
 
-In order to be fully compatible with and maintain the experience of the __Cocos Creator v2.x__ easing system, all the __Cocos Creator v2.x__ functions are transplanted in __Cocos Creator 3.0__.
+![tween-index.png](img/homeImgGame.png)
 
-> **Note**: `action` has been abandoned, please use `tween`.
+Tween systems are widely used in game development, and one of their main purposes is for solving the problem of dynamic animations when offline animations cannot meet the demand.
 
-> **Note**: there is no longer a dependency on `tween.js`. If you use the relevant features of `tween.js`, please adapt.
+In Cocos Creator, tweening can be used for transforming position, rotation, scaling and color in addition to the usual animation information, but also supports delay, queueing, parallelism and other action behaviors.
```

Comment:
In Cocos Creator, tweens can be applied to properties such as transform, rotation, scaling and color, as well as adding additional mechanisms such as delaying, queuing and parallelism.